run_tests.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
This is the only python script where I see /usr/bin/env python3. others have /usr/bin/python.
I think other python scripts should have python 3 as well.

Reply:
Done in 31ea9a9